src/core/utils/Parser.ts

```diff
@@ -89,6 +89,19 @@ function parseElementNode(currentContext: ParsingContext): ParsingContext {
     const context = { ...currentContext };
 
     const node = context.node;
+    // A <br/> with no siblings is there only to make its parent visible.
```

Comment:
@dmo-odoo This is basically your suggestion from your review of https://github.com/odoo-dev/jabberwock/pull/40/commits/038aa5785a2f96bcfbf007f3b469f90707554cce. I did however not include the part that accounts for textual range nodes for commit scope reasons.